sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVOptions.scala

```diff
@@ -117,6 +117,9 @@ class CSVOptions(
 
   val nullValue = parameters.getOrElse("nullValue", "")
 
+  val emptyValueInRead = parameters.getOrElse("emptyValue", "")
```

Comment:
I would just call it `emptyValue` for consistency with other options here.

Reply:
I though that as well. Just for the shake of providing backwards compatibility as we already have in `ignoreLeadingWhiteSpaceInRead` and `ignoreLeadingWhiteSpaceFlagInWrite` I implemented that in that way.
What do you say?

Reply:
I had to name them differently names because the default values are different. We can change the variable later if the same thing should happen

Reply:
I had to name them differently names because the default values are different. Ah, yea then it makes sense here. I rushed to read.